docs/README.md

```diff
@@ -18,6 +19,9 @@
 
 This document gives the overview of all the available documentation for Modelina.
 
+### [Internal language](./Internal%20language.md)
```

Comment:
I see there are several names for the same thing and I think it can confuse the reader: Internal language, internal data model, custom data model. We should stick with one name.

Reply:
You are right, changing!